src/components/categoryFilter/CategoryFilter.jsx

```diff
@@ -0,0 +1,22 @@
+import { Title, List, ListItem, Name, Count } from './styles';
+import CATEGORIES from 'constants/CATEGORIES';
+
+const CategoryFilter = () => {
+  return (
+    <>
+      <Title>Categories</Title>
+      <List>
+        {CATEGORIES.map((category) => {
+          return (
+            <ListItem key={category.name}>
+              <Name>{category.name}</Name>
+              <Count>({category.count})</Count>
```

Comment:
я бы поменял на шаблон
```suggestion
              <Count>{`(${category.count})`}</Count>
```

Reply:
Изменил на шаблон ``{`(${category.count})`}``